pkg/observable/channel/observable_test.go

```diff
@@ -304,39 +307,41 @@ func TestChannelObservable_SequentialProductionAndUnsubscription(t *testing.T) {
 
 			require.Equalf(
 				t, len(expectedNotifications[obsnIdx]),
-				len(*obsrvn.Notifications),
+				len(obsrvn.Notifications),
 				"observation index: %d, expected: %+v, actual: %+v",
-				obsnIdx, expectedNotifications[obsnIdx], *obsrvn.Notifications,
+				obsnIdx, expectedNotifications[obsnIdx], obsrvn.Notifications,
 			)
 			for notificationIdx, expected := range expectedNotifications[obsnIdx] {
 				require.Equalf(
 					t, expected,
-					(*obsrvn.Notifications)[notificationIdx],
+					(obsrvn.Notifications)[notificationIdx],
 					"allExpected: %+v, allActual: %+v",
-					expectedNotifications[obsnIdx], *obsrvn.Notifications,
+					expectedNotifications[obsnIdx], obsrvn.Notifications,
 				)
 			}
 		})
 	}
 }
 
-// TECHDEBT/INCOMPLETE: add coverage for active observers closing when producer closes.
-func TestChannelObservable_ObserversCloseOnProducerClose(t *testing.T) {
-	t.Skip("add coverage: all observers should close when producer closes")
+// TECHDEBT/INCOMPLETE: add coverage for active observers closing when publishCh closes.
```

Comment:
```suggestion
// TODO_TECHDEBT/TODO_INCOMPLETE: add coverage for active observers closing when publishCh closes.
```

Trying to move to a world where we prefix everything with TODO